src/main/java/step2/LottoPrize.java

```diff
@@ -0,0 +1,37 @@
+package step2;
+
+import java.util.Arrays;
+
+public enum LottoPrize implements Comparable<LottoPrize>{
+
+    FIRST(2000000000,new HitCount(6)),
```

Comment:
```suggestion
    FIRST(2_000_000_000,new HitCount(6)),
```
언더스코어를 활용하여 가독성을 높힐 수 있습니다!